scripts/inner/kubestellar-kube-bind

```diff
@@ -61,36 +75,55 @@ then
     exit 1
 fi
 
-space_conf_file="/tmp/$space_name.conf"
-kubectl-kubestellar-get-config-for-space --space-name $space_name --sm-core-config $SM_CONFIG --sm-context $SM_CONTEXT $in_cluster --output $space_conf_file
-KUBECONFIG=$space_conf_file
-
-if kubectl get crd $resource_to_bind.edge.kubestellar.io &> /dev/null; then
-    echo "CRD for $resource_to_bind already in place"
-    exit 0
-fi
+set -e
 
-mkdir -p $kb_dir
-konnector_log_file=$kb_dir/kube-bind-konnector-$space_name.log
+konnector_dir="konnector-${space_name}"
+konnector_log_file="${konnector_dir}/log"
+finally=""
 
 if pgrep konnector | xargs ps e -p 2>/dev/null | egrep "KB_CONSUMER=$space_name(\s|$)" &> /dev/null; then
     echo "kube-bind konnector for space $space_name already running"
+    consumer_kubeconfig=$(mktemp "$space_name".kubeconfig#XXXX)
+    finally="rm -f '$consumer_kubeconfig'; "
+    trap "$finally" EXIT
```

Comment:
Why two traps in a single script ? Can we do conditional cleanup and define a single trap at the beginning ?

Reply:
There is already one conditional (the `if pgre konnector ...` just above), I thought it simpler to not have a more complicated command in the trap.